Sources/Rendering/OpenGL/PolyDataMapper/index.js

```diff
@@ -1866,6 +1869,7 @@ function vtkOpenGLPolyDataMapper(publicAPI, model) {
         colors: c,
         cellOffset: 0,
         vertexOffset: 0, // Used to keep track of vertex ids across primitives for selection
+        useTCoordsPerCell,
         haveCellScalars: model.haveCellScalars,
```

Comment:
No, I did give a look at this variable but it is true whenever cells are used.
I need a flag to know if the tcoords are per cell or per point. It is exactly what is `areScalarsMappingFromCells` for.
For example, when `interpolateScalarsBeforeMapping` is false and we use cell data, then `useTCoordsPerCell` will be false and `haveCellScalars` will be true.

For more information, from the doc of `getAreScalarMappingsFromCells`:
```
When scalars are mapped from cells,
there is one color coordinate per cell instead of one per point
in the vtkDataArray getColorCoordinates().
It means that when getAreScalarsMappedFromCells() is true,
the number of tuples in getColorCoordinates() is the number of points,
and when getAreScalarsMappedFromCells() is false,
the number of tuples in getColorCoordinates() is the number of cells.
```